packages/simulator/streams/orders/ikea.js

```diff
@@ -126,14 +126,20 @@ function read() {
         map((rows) => ({ key: group.key, rows }))
       )
     ),
-    mergeMap(
-      ({ key, rows }) =>
-        search(origins[key].name).then(({ name, position }) =>
-          rows.map((row) => ({ pickup: { name, position }, ...row }))
-        ),
-      1
-    ),
-    mergeAll(1),
+    mergeMap(({ key, rows }) => {
+      // TODO: Figure out a good way to distribute the orders to the distribution centers.
```

Comment:
Egentligen skulle vi mata in fleets.json här så att paketen kan fördelas enligt flottornas inbördes marknadsandelar.